go.mod

```diff
@@ -1,10 +1,18 @@
 module github.com/newrelic/nri-haproxy
 
-go 1.16
+go 1.18
 
 require (
-	github.com/golangci/golangci-lint v1.46.2
 	github.com/newrelic/infra-integrations-sdk v3.7.1+incompatible
 	github.com/stretchr/testify v1.7.1
```

Comment:
Maybe take the opportunity to update also the sdk and testify?

Reply:
done